app/lib/lnd/lightning.js

```diff
@@ -87,57 +98,40 @@ class Lightning {
    */
   async onBeforeConnect() {
     mainLog.info('Connecting to Lightning gRPC service')
-    const { rpcProtoPath, host, cert, macaroon, type } = this.lndConfig
+    const { host, macaroon, type } = this.lndConfig
 
     // Verify that the host is valid before creating a gRPC client that is connected to it.
     return (
       validateHost(host)
         // If we are trying to connect to the internal lnd, wait upto 20 seconds for the macaroon to be generated.
-        .then(() => (type === 'local' ? waitForFile(macaroon, 20000) : Promise.resolve()))
-        // Attempt to connect using the supplied credentials.
-        .then(async () => {
-          // Load the gRPC proto file.
-          // The following options object closely approximates the existing behavior of grpc.load.
-          // See https://github.com/grpc/grpc-node/blob/master/packages/grpc-protobufjs/README.md
-          const options = {
-            keepCase: true,
-            longs: Number,
-            enums: String,
-            defaults: true,
-            oneofs: true
+        .then(() => {
+          if (type === 'local') {
+            return waitForFile(macaroon, 20000)
           }
-          const packageDefinition = await load(rpcProtoPath, options)
+          return
```

Comment:
is there a reason why we have `return` here and in a couple of other places? If we omit return it will still return undefined

Reply:
Our lint rules require an explicit return inside of promise chains.